src/utils/utils.js

```diff
@@ -1,96 +1,97 @@
 const initialCards = [
-  {
-    name: "Yosemite Valley",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lake-louise.jpg",
-  },
-  {
-    name: "Lake Louise",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lake-louise.jpg",
-  },
-  {
-    name: "Bald Mountains",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/bald-mountains.jpg",
-  },
-  {
-    name: "Latemar",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/latemar.jpg",
-  },
-  {
-    name: "Vanoise National Park",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/vanoise.jpg",
-  },
-  {
-    name: "Lago di Braies",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lago.jpg ",
-  },
-];
-
-const avatarButton = document.querySelector('.profile__avatar-button');
-const avatarImage = document.querySelector('.profile__image');
-
-const profileTitleText = document.querySelector(".profile__title");
-const profileDescriptionText = document.querySelector(".profile__description");
-const profileEditBtn = document.querySelector(".profile__edit-button");
-const cardAddBtn = document.querySelector(".profile__add-button");
-
-const profileForm = document.forms["profile-form"];
-const cardForm = document.forms["card-form"];
-
-const editProfileModalTitleInput = document.querySelector(
-  "#profile-title-input"
-);
-const editProfileModalDescriptionInput = document.querySelector(
-  "#profile-description-input"
-);
-
```

Comment:
* I don't see the code of `PopupWithConfirmation.js` in the `Files changed`.  So, I have to write the comments here: please, delete line 11 (method `open`) from there.

*  You shouldn’t create the **same methods** of the parent class if you don’t change it inside and only call it with `super`. The parent methods are available in any inheritor of the class.